centos.org/ansible/fetch_debug_files.yml

```diff
@@ -3,7 +3,7 @@
   tasks:
     - name: 'Find any files from debug'
       find:
-        paths: "/tmp/debug-{{ pipeline_type }}-{{ pipeline_version }}-{{ pipeline_os }}"
+        paths: "/tmp/debug-{{ pipeline_type }}-{{ pipeline_version }}-{{ pipeline_os }}-{{ pipeline_action }}"
```

Comment:
Just for my own sanity: this expands to `/tmp/debug-foreman-nightly-el8-install`.